src/operator/some.ts

```diff
@@ -0,0 +1,69 @@
+import { Operator } from '../Operator';
+import { Observer } from '../Observer';
+import { Observable } from '../Observable';
+import { Subscriber } from '../Subscriber';
+
+/**
+ * Returns an Observable that emits whether or not any items of the source satisfies the condition specified.
+ * @param {function} predicate a function for determining if an item meets a specified condition.
+ * @param {any} [thisArg] optional object to use for `this` in the callback
+ * @return {Observable} an Observable of booleans that determines if all items of the source Observable meet the condition specified.
+ * @method some
+ * @owner Observable
+ */
+/* tslint:disable:max-line-length */
+export function some<T>(this: Observable<T>, predicate: (value: T, index: number, source: Observable<T>) => boolean, thisArg?: any): Observable<boolean> {
```

Comment:
v5 drops support of `thisArg` except native interface has `thisArg` to match it. (https://github.com/ReactiveX/rxjs/issues/878)